projects/TPVFormer/config/tpvformer_8xb1-2x_nus-seg.py

```diff
@@ -0,0 +1,317 @@
+_base_ = ['mmdet3d::_base_/default_runtime.py']
```

Comment:
```suggestion
_base_ = ['../../../configs/_base_/default_runtime.py']
```
better to use the relative path to avoid potential usage trouble.